doc/ref/gappkg.xml

```diff
@@ -923,7 +924,7 @@ before the package is loaded so it will be silently overwritten.
 <Index>needed package</Index>
 <Index>suggested package</Index>
 <Index Subkey="for a GAP package">dependencies</Index>
-It is possible for one &GAP; package <C>A</C>, say,
+It is possible for one &GAP; package, let us say <C>A</C>,
```

Comment:
Or just drop the ", say" without replacement?

Reply:
If it makes sense to remove it I would. I think that makes the text smoother to read.

Reply:
Agreed.